Formula/git-series.rb

```diff
@@ -0,0 +1,20 @@
+class GitSeries < Formula
+  desc "Track changes to a patch series over time"
+  homepage "https://github.com/git-series/git-series"
+  url "https://github.com/git-series/git-series/archive/0.8.10.tar.gz"
+  sha256 "d70c9a2105c69fdd6059c04285ddb74b3b10852d9729dd8a14681b9bf36790be"
+
+  depends_on "cmake" => :build
+  depends_on "rust" => :build
+  depends_on "libssh2"
+
+  def install
+    system "cargo", "build", "--release"
+    bin.install "target/release/git-series"
+    man1.install "git-series.1"
+  end
+
+  test do
+    system bin/"git-series", "-V"
```

Comment:
Can’t we write a test that does something more substantial than e.g. `-V`, `--version`, or `--help`?